### DISCUSSION TITLE
请更新微信群QR 我想进群

### DISCUSSION THREAD

**Discussion by franva** [modelscope/FunClip#119] [General]
想进去学习

**Comment by LauraGPT** [comment]
感谢反馈。微信群二维码可能会过期；README 现在已经增加了长期可用的 GitHub Discussions 入口，无法扫码时可以直接在这里提问、交流或分享项目：<https://github.com/modelscope/FunClip/discussions>。

对应改动已合并：<https://github.com/modelscope/FunClip/pull/196>。微信群二维码仍会在拿到当前有效图片后更新。
